GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -81,6 +82,19 @@ public GitProcess(string gitBinPath, string workingDirectoryRoot)
             }
         }
 
+        public static string ExpireTimeDateString
+        {
+            get
+            {
+                if (expireTimeDateString == null)
+                {
+                    expireTimeDateString = DateTime.Now.Subtract(TimeSpan.FromDays(1)).ToShortDateString();

```

Comment:
Before we chose to expire after 1 hour and now we are after 1 day.  I know that this was never working before and  I don't think that will be an issue because I don't think it should be creating that many commit graphs but wanted to get your thoughts.  Thanks!

Reply:
The important thing is that it doesn't expire things right away, and there are some concerns about timezones and things when it parses the expiry date. So either we get _really specific_ about the time (including timezone offset) or we can just go "at least 24 hours ago, give or take timezone shifts of +/-12 hours".

Since we've been running with the lack of expirations for this long, having a slightly longer expire window will not be a problem.